js/index.js

```diff
@@ -1 +1,68 @@
-// alert("Working"); // Test to see JS is working
```

Comment:
Consider your MVP to be the first iteration of your final product - don't leave tests like this in your final product.